### PR TITLE
Frontend (Fix): Remove unnecessary gitignore inside frontend directory

### PR DESCRIPTION
**Pull request by spyshiv**
* Remove unnecessary `.gitignore` (inside `frontend/`) 
* Remove `frontend/index.html` since `base.html` will build it dynamically
* Add `frontend/index.html` in `.gitignore` (of root directory)

@aka-jain / @deshraj Please Merge this.

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# files/directories to be ignored
-
-.sass-cache/
-
-node_modules/
-bower_components/
-
-dist/
```
Comment (aka-jain) [inline frontend/.gitignore]: @spyshiv We need dist/ I think?

Comment (spyshiv) [inline frontend/.gitignore]: @aka-jain : See the .gitignore of root directory. 
there is `dist/` (line 14) that's working for the `frontend/dist/` .
However run project in locally and see this if everything is fine.

File: .gitignore [hunk 2]
```diff
@@ -79,6 +79,7 @@ fabfile/
 .sass-cache/
 node_modules/
 bower_components/
+frontend/index.html
```
Comment (spyshiv) [inline .gitignore]: @aka-jain : Should i add `frontend/dist/` here to make difference between frontend and backend??

Comment (aka-jain) [inline .gitignore]: Yep, do this please! It will be helpful and more meaningful for other contributors.

**Comment by spyshiv** [comment]
@aka-jain : Done

**Pull request merged**